src/utils/bigNumber.ts

```diff
@@ -1,6 +1,13 @@
 import BigNumber from 'bignumber.js'
+import { ethers } from 'ethers'
+import { SerializedBigNumber } from 'state/types'
 
 export const BIG_ZERO = new BigNumber(0)
 export const BIG_ONE = new BigNumber(1)
 export const BIG_NINE = new BigNumber(9)
 export const BIG_TEN = new BigNumber(10)
+
+export const ethersToSerializedBigNumber = (ethersBn: ethers.BigNumber): SerializedBigNumber =>
+  ethersToBigNumber(ethersBn).toJSON()
```

Comment:
ethers' BigNumber has `toJSON()`.

Reply:
I think the pattern is still to convert to a `BigNumber.js` BN first, and then call `.toJSON()` though. I spoke with @RabbitDoge about this briefly a week or so ago - calling `.toJSON()` directly on the ethers BN can return a different value to the `BigNumber.js` equivalent.